content/docs/iac/adopting-pulumi/migrating-to-pulumi/from-cdk.md

```diff
@@ -0,0 +1,328 @@
+---
+title_tag: "Migrating from AWS CDK"
+meta_desc: Migrate your existing AWS CDK TypeScript application
+title: AWS CDK
+h1: "Migrating from AWS CDK to Pulumi"
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+    iac:
+        name: AWS CDK
+        parent: iac-adopting-migrate
+        weight: 2
+    usingpulumi:
+        identifier: from-aws-cdk
+        parent: migrating
+        weight: 3
+---
+
+If your team has already provisioned infrastructure using AWS CDK, and you'd like to adopt Pulumi, you have two primary strategies you can take:
+
+* [**Coexist**](#referencing-stack-outputs) with resources provisioned by CDK by referencing stack outputs.
+* [**Convert**](#converting-cdk-applications) your CDK application to use [Pulumi CDK](https://github.com/pulumi/pulumi-cdk)
+
+## Referencing Stack Outputs
```

Comment:
This section I copied over from the `From CloudFormation` page since it also applies to CDK. I could instead just have a reference to that page instead of copying over the content. Let me know what you think